src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -62,6 +62,7 @@
  * |-----------------------------------------------------------|----------------------------------------------------------------------------|
  * | Find any tag.                                             | `$tags->next_tag();`                                                       |
  * | Find next image tag.                                      | `$tags->next_tag( array( 'tag_name' => 'img' ) );`                              |
+ * | Find next image tag.                                      | `$tags->next_tag( 'img');`                                                      |
```

Comment:
```suggestion
 * | Find next image tag. (short)                              | `$tags->next_tag( 'img' );`                                                      |
```
A slightly different description will help identify why both are useful. Also added a missing space after between `'` and `)`.

Reply:
side-note: can we correct the spacing of the table syntax that was broken when these were last updated in https://github.com/WordPress/gutenberg/pull/47958

Reply:
Instead of `(short)` maybe we describe this without connoting that there's a name for the syntax? `(without passing the array)`